### ISSUE TITLE
How to use this with Wireguard-UI installation?

### ISSUE THREAD

**Issue by dandud100** [alfiosalanitri/wireguard-client-connection-notification#7]
Hey, i would really like to use this notifier but it does not suite my installation.

A while ago i installed wireguard together with wireguard-ui using this script: `wget https://raw.githubusercontent.com/KleSecGmbH/ioBroker/main/wireguard/installer-pve.sh -O installer-pve.sh && bash installer-pve.sh` from: https://forum.iobroker.net/topic/49177/anleitung-wireguard-mit-wireguard-ui-auf-proxmox

Now i wonder how i can use your notifier because i do not have a /etc/wireguard/*keys*/ directory.

Greetings from Germany ;)

**Comment by alfiosalanitri** [comment]
Hi, have you searched inside the container created by proxmox?

**Comment by dandud100** [comment]
> Hi, have you searched inside the container created by proxmox?

I do not have a clue where to search exactly.

**Comment by alfiosalanitri** [comment]
The /etc/wireguard directory exists? Which files or directories contains?

**Comment by dandud100** [comment]
> The /etc/wireguard directory exists? Which files or directories contains?

It exists but with only wg0.conf and no directories.

**Comment by alfiosalanitri** [comment]
Have you already connected a client to your server with wireguard?

**Comment by dandud100** [comment]
> Have you already connected a client to your server with wireguard?

Yes, I created multiple wireguard profiles through the wireguard UI and I have multiple clients connected to my wireguard server.

**Comment by alfiosalanitri** [comment]
I am sorry but this script cannot work 😞. If I understand correctly, wireguard UI saves client keys in the database. You can see if you can find the keys inside /opt/wireguard-ui

**Comment by alfiosalanitri** [comment]
I can't do much right now, because I have no information about the files and I have no way to install your version.

can you share me a repo with a copy of your wireguard-ui folder?  without copying the keys with sensitive data?  create similar files.

**Comment by dandud100** [comment]
> I can't do much right now, because I have no information about the files and I have no way to install your version.
> 
> can you share me a repo with a copy of your wireguard-ui folder? without copying the keys with sensitive data? create similar files.

Sorry, forgot to answer your question about the files. Did you try to get the files yourself in the meantime?
